### DISCUSSION TITLE
Passive tooltips without touchable overlay?

### DISCUSSION THREAD

**Discussion by wbobeirne** [react-native-elements/react-native-elements#3861] [Q&A]
Hey there, I'm implementing an app which has tooltips for onboarding you to new features you haven't used yet, but I don't want it them to block the user from interacting with the rest of the app. I only want the tooltip to dismiss once you've interacted with the element it's pointing at, but you should be able to ignore it and tap other things on the screen.

Is this possible using the existing props in `<Tooltip />`? I wouldn't mind making my own component, but there is a _ton_ of logic for Tooltip around positioning that I don't want to miss out on.

**Comment by ryanjsfx2424** [comment]
Bump.
